CalibTracker/SiStripChannelGain/plugins/SiStripGainFromCalibTree.cc

```diff
@@ -196,6 +193,7 @@ class SiStripGainFromCalibTree : public ConditionDBWriter<SiStripApvGain> {
         enum statistic_type {None=-1, StdBunch, StdBunch0T, FaABunch, FaABunch0T, IsoBunch, IsoBunch0T, Harvest};
 
         std::vector<string>    dqm_tag_;
+        std::vector booked_dir_;
```

Comment:
<head slap> helps when one compiles the code before doing the pull request</head slap>